diff -up a/dwm.c b/dwm.c
--- a/dwm.c	2020-03-31 23:57:44.211919827 -0300
+++ b/dwm.c	2020-04-01 00:01:21.501924665 -0300
@@ -797,7 +797,7 @@ drawbar(Monitor *m)
 	Client *c;
 
 	/* draw status first so it can be overdrawn by tags later */
-	if (m == selmon) { /* status is only drawn on selected monitor */
+	if (m == selmon || 1) { /* status is only drawn on selected monitor */
 		sw = m->ww - drawstatusbar(m, bh, stext);
 	}
 
@@ -2307,9 +2307,11 @@ updatesizehints(Client *c)
 void
 updatestatus(void)
 {
+	Monitor* m;
 	if (!gettextprop(root, XA_WM_NAME, stext, sizeof(stext)))
 		strcpy(stext, "dwm-"VERSION);
-	drawbar(selmon);
+	for(m = mons; m; m = m->next)
+		drawbar(m);
 	updatesystray();
 }
 
