diff -up dwm-6.2/dwm.c dwm-6.2_wtitlebg/dwm.c
--- dwm-6.2/dwm.c	2019-06-16 01:02:51.347627507 -0400
+++ dwm-6.2_wtitlebg/dwm.c	2019-06-16 01:06:33.263096241 -0400
@@ -870,7 +870,7 @@ drawbar(Monitor *m)
 
 	if ((w = m->ww - sw - stw - x) > bh) {
 		if (m->sel) {
-			drw_setscheme(drw, scheme[m == selmon ? SchemeSel : SchemeNorm]);
+			drw_setscheme(drw, scheme[SchemeNorm]);
 			drw_text(drw, x, 0, w, bh, lrpad / 2, m->sel->name, 0);
 			if (m->sel->isfloating)
 				drw_rect(drw, x + boxs, boxs, boxw, boxw, m->sel->isfixed, 0);
